shared/src/commonMain/kotlin/com/fbada006/shared/icons/InkEraser.kt

```diff
@@ -9,56 +24,56 @@ import androidx.compose.ui.graphics.vector.ImageVector
 import androidx.compose.ui.graphics.vector.path
 import androidx.compose.ui.unit.dp
 
-val InkEraser: ImageVector
-	get() {
-		if (ink_eraser != null) {
-			return ink_eraser!!
-		}
-		ink_eraser = ImageVector.Builder(
+val inkEraser: ImageVector
```

Comment:
There is a naming inconsistency here. In this class, you have changed this to lower case but not the rest of the icons. Why is this the case? Is this a mistake?

Reply:
Spotless was throwing an error before due to the previous Casing Style (Pascal Case) so I changed it to Camel Case to resolve it. If that is not important please let me know so that I can revert the changes...

Reply:
I have reverted the changes which means that `spotless` will keep throwing an error here (Property name should start with a lowercase letter and use camel case). The alternative that I had implemented before was disabling `ktlint_standard_property-naming` but this would have impacted the rest of the code so I did away with it...